.travis.yml

```diff
@@ -56,6 +56,7 @@ jobs:
       script: true
       before_deploy:
         - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
+        - lerna run typedoc
```

Comment:
When we create the typedocs before the deploy stage, we should add the `docs` folder to `.npmignore` to prevent it from publishing to npm.

Reply:
Never mind.. the docs aren't created in the packages... The `.npmignore` in the root level confused me